lib/js/components/Layout/Layout.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+	<div class="layout">
+		<div v-if="sideMenuVisible" class="layout__navigation">
+			<div class="layout__mainMenu">
+				<slot name="mainMenu" />
+			</div>
+			<div class="layout__sideNav">
+				<slot name="sideNav" />
+			</div>
```

Comment:
did UX agree on separating left column into `mainMenu` and `sideNav`? How do we plan to implement `AssessmentPlanEditor` when `mainMenu` has fixed width?